src/client/sandbox/event/listeners.js

```diff
@@ -296,4 +296,18 @@ export default class Listeners extends EventEmitter {
             eventListeningInfo.outerHandlersWrapper = wrapper;
         }
     }
+
+    getEventListeners (el, event) {
+        const eventCtx = this.listeningCtx.getEventCtx(el, event);
+
+        if (!eventCtx)
+            return null;
+
+        const outerHandlers = [];
+
+        for (let i = 0; i < eventCtx.outerHandlers.length; i++)
```

Comment:
use the `Array.map` function. 
See example of the array' methods using - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/event-emitter.js#L3